packages/mui-material/src/CardHeader/CardHeader.js

```diff
@@ -192,13 +222,110 @@ CardHeader.propTypes /* remove-proptypes */ = {
    * @default false
    */
   disableTypography: PropTypes.bool,
+  /**
+   * The props used for each slot inside.
+   * @default {}
+   */
+  slotProps: PropTypes.shape({
```

Comment:
Shouldn't `slotProps` have a similar structure to:

https://github.com/mui/material-ui/blob/03325351a4c3a7b1ef9adbb7ad49a5caa39559a9/packages/mui-material/src/Tooltip/Tooltip.js#L954-L959

I think all (or at least most) `slotProps`'s prop types follow this.

Reply:
Update the slotProps type fixes this.